toolchain/check/context.cpp

```diff
@@ -505,6 +532,102 @@ auto Context::ImplicitAsRequired(Parse::Node parse_node, SemIR::NodeId value_id,
   return output_value_id;
 }
 
+static auto CanConvertTupleToArray(Context& context, SemIR::Node tuple_type,
```

Comment:
```suggestion
// Returns true if the tuple could be converted to the array type.
static auto CanConvertTupleToArray(Context& context, SemIR::Node tuple_type,
```

Cursory comment request.

Reply:
Obsolete after #3208.